lib/main.dart

```diff
@@ -12,10 +13,14 @@ import 'package:thc/views/settings/settings.dart';
 import 'package:thc/views/survey/survey_questions.dart';
 import 'package:thc/views/survey/survey_screen.dart';
 import 'package:thc/views/survey/survey_theme.dart';
+import 'firebase_options.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await loadFromLocalStorage();
+  await Firebase.initializeApp(
+    options: DefaultFirebaseOptions.currentPlatform,
+  );
```

Comment:
```suggestion
  final storage = loadFromLocalStorage();
  final firebase = Firebase.initializeApp(options: DefaultFirebaseOptions.currentPlatform);
  await storage;
  await firebase;
```

Really small change that can make the app launch a tiny bit faster—this allows `firebase` to initialize while we're waiting for `storage` to finish.